src/iv/imageviewer.cpp

```diff
@@ -595,7 +593,7 @@ ImageViewer::readSettings (bool ui_is_set_up)
     linearInterpolationBox->setChecked (settings.value ("linearInterpolation").toBool());
     darkPaletteBox->setChecked (settings.value ("darkPalette").toBool());
     QStringList recent = settings.value ("RecentFiles").toStringList();
-    BOOST_FOREACH (const QString &s, recent)
+    for (auto&& s : recent)
```

Comment:
Did you find a good set of guidelines on when to use `auto` vs `auto&` vs `auto&&` vs any of the `const` prefixed variants? Its one aspect of C++11 I'm not 100% clear on yet.